### DISCUSSION TITLE
Directus default credentials for platform.sh

### DISCUSSION THREAD

**Discussion by denniszervas** [directus/directus#3416] [Archive]
Hi all, 

I installed Directus using platform.sh
I cannot login, don't know what the default credentials are. 
Any help is appreciated! 

thank you 

Dennis

**Comment by rijkvanzanten** [comment]
@chadwcarlson Do you any insight on this? 🙂

**Comment by chadwcarlson** [comment]
Sure do - @denniszervas the initial admin credentials are located in the [`.environment`](https://github.com/platformsh-templates/directus/blob/c1d7988ba6fca09db27ce916e80992b0e1f32496/.environment#L2) file - same credentials as the main Directus demo.
